ims/common/config.py

```diff
@@ -9,53 +9,94 @@
 __config = None
```

Comment:
Right but do you need a global variable for that is the question. There are many ways to do this but again this is nit, if you have a strong preference of using global, I am ok with that.